### PR TITLE
Implements the front end for sorting

### PR DESCRIPTION
**Pull request by rossjones**
Makes the appropriate headings (in summary only for now) clickable so
they can sort the page of results.

### CODE REVIEW

File: app/helpers/sort_helper.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  def display_arrow(field_name)
+    uri = URI.parse(request.original_url)
+
+    query = Rack::Utils.parse_query(uri.query)
+    current_sort = query['sort']
+    return '' unless current_sort.present? && current_sort.start_with?(field_name)
+
+    if current_sort.end_with?('asc')
+      '<span class="sort-arrow up">&#9650;</span>'.html_safe
```
Comment (TobyRet) [inline app/helpers/sort_helper.rb]: `content_tag(:span, class: "sort-arrow up", "&#9650")`

Comment (TobyRet) [reply app/helpers/sort_helper.rb]: Using this the `content_tag` helper takes care of html_safe for you

Comment (TobyRet) [inline app/helpers/sort_helper.rb]: `content_tag(:span, '&#9650'.html_safe, class: "sort-arrow up" )`

Comment (rossjones) [reply app/helpers/sort_helper.rb]: Thanks

**Pull request merged**